### DISCUSSION TITLE
CS:APP 第三章学习笔记

### DISCUSSION THREAD

**Discussion by giscus** [ouuan/iles-blog#136] [Comments]
# CS:APP 第三章学习笔记

CS:APP 第三章 “Machine-Level Representation of Programs” 的学习笔记。
这章的主要内容为汇编（machine-level programming）。

https://ouuan.moe/post/2022/09/csapp-3

**Comment by Cookiexs** [comment]
不错👍，❤️爱来自瓷器！
